demos/demo_runner/iot_demo_freertos.c

```diff
@@ -344,14 +344,14 @@ void runDemoTask( void * pArgument )
         #ifdef democonfigMEMORY_ANALYSIS
             /* If memory anaalysis is enabled metrics regarding the heap and stack usage of the demo will print. */
             /* This format is used for easier parsing and creates an avenue for future metrics to be added. */
-            xAfterDemoHeapSize = demoMEMORY_ANALYSIS_MIN_EVER_HEAP_SIZE();
-            IotLogInfo( "memory_metrics:freertos_heap:total:bytes:%u", demoMEMORY_ANALYSIS_HEAP_SIZE );
+            xAfterDemoHeapSize = democonfigMEMORY_ANALYSIS_MIN_EVER_HEAP_SIZE();
```

Comment:
chatted offline.

Given new heap metrics is coming, https://sourceforge.net/p/freertos/code/HEAD/tree/trunk/FreeRTOS/Source/include/portable.h#l132, needs consideration on the variable naming convention and database column name convention. (Trying to prevent people using the wrong data column for querying, and trying to prevent a column in database getting repurposed.)

Reply:
Agreed. Once kernel is updated in AFR I will bump schema version and adjust names